mmv1/overrides/terraform/property_override.rb

```diff
@@ -39,103 +39,7 @@ class PropertyOverride < Overrides::PropertyOverride
       # Collection of fields allowed in the PropertyOverride section for
```

Comment:
Does the `Validation` class above still need to exist? I _think_ it does- current yamls reference its type. If it does we could clean that up in a followup independently of migrating those products, right?

Reply:
Yes, there should be a "final cleanup" once all products have been migrated which includes removing overrides as a whole.